xl2times/transforms.py

```diff
@@ -2212,7 +2285,11 @@ def eval_and_update(
         # TFM_UPD: expand wildcards in each row, query FI_T to find matching rows,
         # evaluate the update formula, and add new rows to FI_T
         # TODO perf: collect all updates and go through FI_T only once?
-        for _, row in updates.iterrows():
+        for _, row in tqdm(
+            updates.iterrows(),
+            total=len(updates),
+            desc=f"Processing wildcard for {datatypes.Tag.tfm_upd}",
+        ):
```

Comment:
Added progress bars for slow operations here.  will remove these once I get this function optimised also.